edgedb/datatypes/namedtuple.c

```diff
@@ -58,62 +60,34 @@ EdgeNamedTuple_New(PyObject *desc)
         return NULL;
     }
 
-    EdgeNamedTupleObject *nt = NULL;
-    EDGE_NEW_WITH_FREELIST(EDGE_NAMED_TUPLE, EdgeNamedTupleObject,
-                           &EdgeNamedTuple_Type, nt, size);
+    PyTupleObject *nt = NULL;
+    EDGE_NEW_WITH_FREELIST(EDGE_NAMED_TUPLE, PyTupleObject, type, nt, size);
```

Comment:
We cannot reuse the builtin freelist for `tuple` because the `ob_type` is not `&PyTuple_Type`.